src/test/java/team4/footwithme/member/repository/MemberRepositoryTest.java

```diff
@@ -0,0 +1,33 @@
+package team4.footwithme.member.repository;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import team4.footwithme.member.domain.*;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest
+class MemberRepositoryTest {
```

Comment:
제가 IntegerationTestSupport라는 추상클래스를 만들어두었는데 이거 활용하셔서 `@SpringBootTest` 어노테이션 지우고 사용하셔도 좋아요

```suggestion
@Transactional
class MemberRepositoryTest extends IntegrationTestSupport {
```

`@SpringBootTest`를 모든 테스트마다 붙여서 사용하면 테스트할때마다 스프링부트가 실행되어서 느려진다고 하더라구요 환경을 통합해서 부트를 한번만 키는 방식으로 변경해도 좋을 것 같아요
그리고 트랜잭셔널 붙인건 위에 얘기한거처럼 롤백을 위해서 사용하는게 편리해서 넣었답니다

[[Spring] 테스트 환경](https://tommykim.tistory.com/87) 이 글 참고해보시면 좋을 것 같아요

Reply:
아하 다른 테스트도 다 지워서 사용하면 되는거죠?

Reply:
네네 컨트롤러를 제외한 테스트에서는 상속받아서 사용하면 됩니다 ! 
컨트롤러는 컨트롤러서포트를 상속받아야해서 조금 다르게 사용하시면 됩니다

Reply:
넵 알겠습니다